xbmc/video/dialogs/GUIDialogCMSSettings.cpp

```diff
@@ -0,0 +1,243 @@
+/*
+ *      Copyright (C) 2005-2014 Team XBMC
```

Comment:
Copyright (C) 2016 - your name - then exchange everything that points to XBMC in the rest of the header to Kodi and kodi.tv - in case you don't want to put your name in there you can also set copyright to Team Kodi